src/Templates/src/templates/maui-resourcedictionary-xaml/.template.config/ide.host.json

```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "http://json.schemastore.org/vs-2017.3.host",
+    "appliesTo": "UseMaui",
+    "defaultItemExtension": "xaml",
+    "itemHierarchyPaths": [ ".NET MAUI" ],
+    "icon": "ide/MauiContentViewXamlFile.ico",
```

Comment:
Could we use a different icon?

Reply:
Very good! Any clue where we get that? :D